service/crawlingTitleService.js

```diff
@@ -2,27 +2,47 @@ const puppeteer = require("puppeteer");
 
 const getCrawlingTitle = async (req, res) => {
   const decodedLink = decodeURIComponent(req.params.url);
+  const keyword = req.query.keyword;
   const browser = await puppeteer.launch({ headless: true });
 
   try {
     const page = await browser.newPage();
-
     await page.goto(decodedLink);
 
-    const element = await page.$("title");
-    const title = await page.evaluate(
-      (element) => element.textContent,
-      element
-    );
+    const title = await page.$eval("title", (element) => element.textContent);
+    const hasTitleKeyword = title.includes(keyword);
+
+    if (!title) {
+      return res.status(200).send({ message: `[This Title does not exist]` });
+    }
 
     return res.status(200).json({
-      title,
+      hasKeyword: hasTitleKeyword,
+      urlTitle: title,
     });
   } catch (error) {
-    return res.status(500).send({ message: `[ServerError occured] ${error}` });
+    if (isCheckTrueThisUrl(decodedLink) === false) {
```

Comment:
```suggestion
    if (!isCheckTrueThisUrl(decodedLink)) {
```
해당 코드도 위의 코드와 마찬가지로  부정연산자를 사용하면 더 좋을 것 같습니다!